app/templates/components/conversations/conversation-thread.hbs

```diff
@@ -1,20 +1,20 @@
 <div class="conversation-thread-inner">
   <div class="conversation-thread__messages">
     {{#if conversation.message.isLoaded}}
-      {{conversations/conversation-part
+      {{conversations/conversation-part-comment
         author=conversation.message.author
         body=conversation.message.body
         readAt=conversation.message.readAt
         sentAt=conversation.message.insertedAt
       }}
     {{/if}}
-    {{#each conversation.conversationParts as |conversationPart|}}
-      {{#if conversationPart.isLoaded}}
-        {{conversations/conversation-part
-          author=conversationPart.author
-          body=conversationPart.body
-          readAt=conversationPart.readAt
-          sentAt=conversationPart.insertedAt
+    {{#each conversation.conversationPartComments as |conversationPart|}}
+      {{#if conversationPartComment.isLoaded}}
+        {{conversations/conversation-part-comment
+          author=conversationPartComment.author
+          body=conversationPartComment.body
+          readAt=conversationPartComment.readAt
+          sentAt=conversationPartComment.insertedAt
```

Comment:
So what you probably wanna do here is keep the `{{#each conversation.conversationParts as |conversation_part|}}` and then do a couple of `if-else` chains inside the `{{#if conversationPart.isLoaded}}`

Think of this way. A conversation has a bunch of conversation parts and we want to render all of them in the order we got them in.

Then, for each of them, if it's a comment-type conversation part, render it as a comment type. If it's a closed-type conversation part, render is as a closed type, etc.

```Handlebars
{{#each conversation.conversationParts as |conversationPart|}}
  {{#if conversationPart.isLoaded}} 
    {{#if (eq conversationPart.partType 'closed')}}
      {{conversations/conversation-part-closed ..}}
    {{else}}
      {{conversations/conversation-part-comment ...}}
    {{/if}}
  {{/if}}
{{/each}}
```

Note that in the example, the comment type is sort of considered the default - the thing we render if it's not any other type. It's not perfect and really, if we do not support all the types our API supports, we have a bug, but at least this will render something, which is preferable to and easier to detect as an issue than nothing.

There may be more elegant ways to do this, but for now, this is good enough. Then, when we add another conversation part type, such as `conversation-part-open`, we would add an `else if` here, between the `if` and the `else`.